src/isomorphic/classic/element/__tests__/ReactElementValidator-test.js

```diff
@@ -84,37 +84,81 @@ describe('ReactElementValidator', function() {
     var Anonymous = React.createClass({
       displayName: undefined,
       render: function() {
-        return <div />;
+        return React.createElement('div');
```

Comment:
Instead of changing these to not use JSX, can you just make the tests like I have in other files, replacing the filename and line number with `(**)`?